lib/Message.mjs

```diff
@@ -1,47 +1,61 @@
+import util from '../util/index.js';
+
 class Argument {
```

Comment:
The Argument class is not a singleton. This is a non-issue due to it not being exported, but this class declaration should probably be in the `else` statement to reduce confusion